sdk/go/common/util/cmdutil/console.go

```diff
@@ -373,8 +373,7 @@ func (m readConsoleModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		// it's time to stop the bubbletea loop.
 		//
 		// Only Enter is considered a success.
-		switch msg.Type {
-		case tea.KeyEnter, tea.KeyCtrlC, tea.KeyEsc:
+		if msg.Type == tea.KeyEnter || msg.Type == tea.KeyCtrlC || msg.Type == tea.KeyEsc {
```

Comment:
This and sdk/go/common/resource/plugin/provider_plugin.go switches did look nicer as switches rather than == == ==. Wonder if we leave 'default' as allowed and then just have something like "default: // ignore other keys" instead?

Reply:
I think we could just annotate this with //nolint:exhaustive as well if we want to keep the switches.  

I shied away from having default as allowed, since it would be quite a bit of work to get rid of all the "default: contract.Fail(...)" situations we have, but it would definitely be doable.